spec.md

```diff
@@ -1086,19 +1086,9 @@ For success responses, the following fields are defined:
 
 | Response Field | Type | Description |
 | --- | --- | --- |
-| dashboard_url | string | The updated URL of a web-based management user interface for the Service Instance; we refer to this as a service dashboard. The URL MUST contain enough information for the dashboard to identify the resource being accessed (`0129d920a083838` in the example below). Note: a Service Broker that wishes to return `dashboard_url` for a Service Instance MUST return it with the initial response to the update request, even if the service is updated asynchronously. If present, MUST be a non-empty string. |
+| dashboard_url | string | The updated URL of a web-based management user interface for the Service Instance; we refer to this as a service dashboard. The URL MUST contain enough information for the dashboard to identify the resource being accessed (`0129d920a083838` in the example below). Note: a Service Broker that wishes to return `dashboard_url` for a Service Instance it MUST return it with the initial response to the update request, even if the service is being updated asynchronously. If present, MUST be a non-empty string. |
```

Comment:
I think the added "it" before "MUST return" isn't correct. Unless you also want to add something like "When" to the start of the sentence.

Reply:
Oops - not sure what happened in my brain there! fixing